packages/common/src/ui/components/inputs/Dropdown/Dropdown.tsx

```diff
@@ -1,16 +1,92 @@
 import React, { FC } from 'react';
-import Select, { SelectProps } from '@material-ui/core/Select';
-import MenuItem, { MenuItemProps } from '@material-ui/core/MenuItem';
-import { styled } from '@material-ui/system';
+import {
+  MenuItem,
+  MenuItemProps,
+  FormControl,
+  Select,
+  InputLabel,
+  outlinedInputClasses,
+  Box,
+  SvgIconProps,
+  ListItemText,
+  styled,
+} from '@material-ui/core';
 
-export const DropdownItem: FC<MenuItemProps> = props => {
-  return <MenuItem {...props} />;
+import { ChevronDown } from '../../../icons';
+
+interface DropdownItemProps extends MenuItemProps {
+  IconComponent?: React.JSXElementConstructor<SvgIconProps>;
+  inset?: boolean;
+}
+
+export const DropdownItem: FC<DropdownItemProps> = props => {
+  return (
+    <MenuItem sx={{ fontSize: '10' }} {...props}>
+      {props.IconComponent ? (
+        <Box mr={1}>
+          <props.IconComponent fontSize="inherit" />
+        </Box>
+      ) : null}
+      <ListItemText
+        inset={props.inset}
+        primaryTypographyProps={{ sx: { fontSize: 'inherit' } }}
+        sx={{ '&.MuiListItemText-inset': { paddingLeft: '22px' } }}
+      >
+        {props.children}
+      </ListItemText>
+    </MenuItem>
+  );
 };
 
-const StyledSelect = styled(Select)({});
+const StyledSelect = styled(Select)(({ theme }) => ({
+  borderRadius: '8px',
+  width: 160,
```

Comment:
Again restrictive 🤔  probably not ideal for the width and maybe should be a min width instead. Then I thought that maybe using a `size` or `width` prop which was an enum would be better and then I gave up

Reply:
wondered similar.  leave for now though - looks great and there's plenty in this PR already